sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/TransactionOverview.kt

```diff
@@ -5,6 +5,9 @@ package cash.z.ecc.android.sdk.model
  *
  * Note that both sent and received transactions will have a positive net value.  Consumers of this class must check
  * [isSentTransaction] if displaying negative values is desired.
+ *
+ * Pending transactions are identified by a null [minedHeight].  Pending transactions are considered expired if the
+ * last synced block exceeds the [expiryHeight].
```

Comment:
We probably want a `isPending` helper

Reply:
I avoided adding isPending here because I think it needs to know the latest scanned height, not the latest height of the blockchain.

Although I could add a method with a parameter for lastScannedHeight, I have a concern that if scanning becomes out-of-order, it isn't clear that clients would be able to pass in the right argument to determine this.

Reply:
If we invert the problem, I wonder if an `isConfirmed` flag is better than `isPending`?

We still lack the information here to report confirmation in this object as it is, but perhaps that could be injected by the synchronizer doing a map on the flow that gets returned.

Reply:
We probably want to disambiguate what Pending and Unconfirmed mean to us. They previously were meshed together and this is a great opportunity to do so.

We already have a `expired_unmined` column. So we can call `Pending` to transactions that have not been mined and have not expired either. 

Then `Unconfirmed` is a little bit more complicated because we need an "anchor" value to say N transactions have passed since this transaction was mined. Information that the "v_transaction" view does not bring with it.